### PR TITLE
De able to deploy the project based on make

### LINKED ISSUE #1190: Update the deploy to Make
Without new build, something like
```
make -f <host> remove-template-files
make -f <host> generate-template-files
```

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1190

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/deploy/hooks/post-restore-code [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -ex
+
+PROJECT_NAME=$1
+CODE_DIR=$2
+
+cd $CODE_DIR
+make -f $TARGET.mk template-clea
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/create/deploy/hooks/post-restore-code]: template-clean?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 2]
```diff
@@ -261,6 +261,8 @@ help:
 	@echo "- compile-py-catalog		Compile the angular translation catalog"
 	@echo "- flake8					Run flake8 checker on the Python code"
 	@echo "- lint-ngeo				Check the JavaScript code with linters for ngeo"
+	@echo "- template-clean		 Clean the template file"
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Shouldn't the indent be fixed in this section? (Command descriptions should be aligned?)

File: c2cgeoportal/scaffolds/create/deploy/hooks/post-restore-code (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/deploy/hooks/post-restore-code]: oups ...

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: tabs crazy ...

**Comment by asaunier** [comment]
There's a typo in the commit title :)
"De able" => "Be able"

**Pull request merged**